src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -61,6 +61,41 @@ public static bool IsZipDeployment(this IEnvironment environment)
                 !string.IsNullOrEmpty(environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteRunFromPackage));
         }
 
+        public static bool IsAppServiceWindowsEnvironment(this IEnvironment environment)
+        {
+            return environment.IsAppServiceEnvironment()
```

Comment:
Wouldn't it be sufficient to check if `IsAppServiceEnvironment` and `RuntimeInformation.IsOSPlatform(OSPlatform.Windows)`?

Reply:
yes, that is correct. I was looking for a better way to detect this. Wasn't aware there is an API like this.